odltools/karaf/dump.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2018 Red Hat, Inc. and others. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+
+line_num = 0
+OPENER = '[{<'
+CLOSER = ']}>'
+INDENT = '  '
+SKIP_FIELDS = 4
+
+
+def dump_karaf_log(args):
+    path = args.path
```

Comment:
why not use the isexists()?

also use the with open() as fp form so we get the close for free.

Reply:
done